docs/usage/self-hosted-configuration.md

```diff
@@ -292,6 +292,11 @@ Like this:
 
 ## dryRun
 
+"extract": Performs extract only, prints results, exits.<br>
+"lookup": Performs extract and lookup, but exits before running through branches.<br>
+"full": Perform a dry run by logging messages instead of creating/updating/deleting branches and PRs.<br>
+Default value = null.
```

Comment:
```suggestion
You can choose from the following behaviors for the `dryRun` config option:

- `null`: REPLACE WITH EXPLANATION, this is the default value
- `"extract"`: Performs extract only, prints results, exits
- `"lookup"`: Performs extract and lookup, but exits before running through branches
- `"full"`: Performs a dry run by logging messages instead of creating/updating/deleting branches and PRs.
```

Changes:

- Convert to list
  - Make `null` default the top list item
- Small rewrite so that the list items start with `performs`
- Add `monospaced` font to the config option strings
- Remove `<br>` HTML code

After applying my suggestion, please replace the `REPLACE WITH EXPLANATION` text with a proper explanation of what `null` means. 😉